messages/id.json

```diff
@@ -0,0 +1,317 @@
+{
+  "titles" : {
+    "privacyPolicy": "Kebijakan Privasi",
+    "terms": "Syarat dan Ketentuan",
+    "pdfSnake": "PDF Snake",
+    "pdfSnakeVersion": "PDF Snake {version}"
+  },
+  "topNav" : {
+    "home": "Beranda",
+    "about": "Tentang",
+    "signIn": "Masuk",
+    "myAccount": "Akun Saya"
+  },
+  "dropTarget" : {
+    "dropHere": "Tarik sebuah file PDF ke sini atau",
+    "browse": "Pilih File untuk Memulai",
+    "docsStayPrivate": "Dokumen-dokumen Anda tetap terjaga kerahasiaannya karena mereka tidak pernah meninggalkan perangkat Anda.",
+    "help": "Bantuan"
+  },
+  "about" : {
+    "privacy": "Baca {link} kami.",
+    "terms": "Dengan menggunakan situs web ini, Anda setuju dengan {link} kami.",
+    "plugIn": "PDF Snake 6 adalah versi daring baru dari <i>Plug-In</i> Adobe Acrobat yang terkenal, {link}.",
+    "questions": "Kami senang menerima pertanyaan dan komentar Anda.",
+    "copyright": "Hak Cipta 2002-{year} Rennie Glen Software LLC"
+  },
+  "choose": {
+    "whatNext": "Apa selanjutnya?"
+  },
+  "tools": {
+    "booklet": "Buklet",
+    "grid": "<i>Grid</i>",
+    "monkey": "<i>Monkey</i>",
+    "resize": "Ubah Ukuran",
+    "crop": "Potong",
+    "rotate": "Putar",
+    "shuffle": "Acak"
+  },
+  "pageSize": {
+    "paperSize": "Ukuran Kertas",
+    "custom": "Kustom",
+    "edit": "Tambah & edit ukuran kertas",
+    "width":  "Lebar",
+    "height": "Tinggi",
+    "landscape": "Lanskap",
+    "help": {
+      "introHtml": "Ukuran kertas untuk dokumen hasil. Untuk menentukan ukuran kertas yang tidak ada dalam pilihan, pilih <strong>Kustom</strong> di kotak <strong>Ukuran Kertas</strong>, lalu masukkan dimensi halaman ke dalam kotak <strong>Lebar</strong> dan <strong>Tinggi</strong>.",
```

Comment:
Notice how the name for this string is "introHtml".  The _Html_ in the name means that in can contain html tags.  I see you used the `<strong>` tag here.  That's great.